### ISSUE TITLE
Documentation wrong fpr return type SelectView::select

### ISSUE THREAD

**Issue by worikgh** [gyscos/Cursive#268]
It returns std::rc::Rc not alloc::rc::Rc as stated at https://docs.rs/cursive/0/cursive/views/struct.SelectView.html#method.selection

**Comment by gyscos** [comment]
Good catch! Seems to be a rustdoc issue. `std::rc::Rc` _is_ `alloc::rc::Rc`, but exported under a different name. I suppose the link should point to `std::rc::Rc` since that's what's imported.
